client/src/Helpers/handle-channel-updates.ts

```diff
@@ -86,11 +86,8 @@ const handleNonInfoMessage = (data: SetWargameMessage, channel: string, message:
       const present = theChannel.messages.some((msg: MessageChannel) => msg._id === message._id)
       if (!present) {
         // chat messages need to go at the end, not the start
-        if (theChannel.cData.channelType === CHANNEL_CHAT) {
-          theChannel.messages.push(newObj)
-        } else {
-          theChannel.messages.unshift(newObj)
-        }
+        theChannel.messages.push(newObj)
```

Comment:
Hello @lilitkarapetyan - I'm nervous about the impact of this change across all of the channels that use this logic.  Could you check please new messages get added properly to each type of channel (including mapping).